history/characters/16000_gnome.txt

```diff
@@ -9,6 +9,7 @@
 	trait=education_learning_4
```

Comment:
Check `Crusader Kings III\logs\error.log` errors. Look for errors mentioning `history\characters`.

Reply:
Resolved all errors returned from search of "make_important_lore_characters_effect" and "important". Other history/characters errors unrelated.